RibbonPhotos/Helpers/Constants.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  Constants.swift
+//  RibbonPhotos
+//
+//  Created by Julia Romanenko on 29.12.2022.
+//
+
+import Foundation
+
+let AccessKey = "X1GaI9onDFAezERyuwGOfq4lezFJvZ7l78iW2YkJdyQ"
+let SecretKey = "MML8fojUnCLRTULAWzMWna99Ba8P38ZwUiv9qS178UM"
+let RedirectURI = "urn:ietf:wg:oauth:2.0:oob"
+let AccessScope = "public+read_user+write_likes"
+let DefaultBaseURL = URL(string: "https://api.unsplash.com")
+let UnsplashAuthorizeURLString = "https://unsplash.com/oauth/authorize"
```

Comment:
Обычно глобальный контекст не принято использовать, такие константы упаковывают в enum 

например так 

```suggestion
enum Constants {
static let accessKey = "X1GaI9onDFAezERyuwGOfq4lezFJvZ7l78iW2YkJdyQ"
...
}
```

обращу внимание, что константы стоит тоже объявлять в camelCase